File: problems/Rhombuses/submissions/time_limit_exceeded/rhombuses.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n int
	fmt.Scanf("%d", &n)

	result := 0
	// summation of i^2 from 1 to n
	for i := 1; i <= n; i++ {
		result += int(math.Pow(float64(i), float64(2)))
	}
	fmt.Println(result)
}

File: problems/Rhombuses/submissions/time_limit_exceeded/rhombuses.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#include <math.h>

int main(int argc, char const *argv[])
{

    long n;
    scanf("%ld", &n);
    long result = 0;
    for (int i = 0; i <= n; i++)
    {
        result += pow(i, 2);
    }

    printf("%ld\n", result);
    return 0;
}
